util/pytest_log_filter.py

```diff
@@ -0,0 +1,16 @@
+import re
+import sys
+
+
+def filter_test_log(log):
```

Comment:
can you add a docstring explaining what log is and what type? Is is the path to the log file?